homeassistant/components/huawei_lte/__init__.py

```diff
@@ -80,21 +81,26 @@ def update(self) -> None:
         self._update()
 
     def _update(self) -> None:
+        from huawei_lte_api.exceptions import \
```

Comment:
3rd party imports can be moved to the top of the module.

Reply:
Ah, done, as well as for the couple of other ones.